softdev.bib

```diff
@@ -2148,3 +2148,12 @@ @book{gudeman1995representing
     month = Oct,
     publisher= {University of Arizona, Tucson}
 }
+
+ - year is from first git commit https://github.com/chromium/octane
+@misc{manish2015rustgc,
+    title = {rust-gc},
+    howpublished = {\href{https://github.com/Manishearth/rust-gc/}{https://github.com/Manishearth/rust-gc/}},
```

Comment:
I think there's a `\link` (or similarly named) macro which avoids the need to repeat information. You might want to check for other web references in the bib file: I think we often also reduce the font size.

Reply:
Okay, I've used this. Not sure what you mean by reducing font size. This is something you do in the .ltx document ?